tokio-util/src/codec/framed_impl.rs

```diff
@@ -7,7 +7,7 @@ use tokio::io::{AsyncRead, AsyncWrite};
 use bytes::BytesMut;
 use futures_core::ready;
 use futures_sink::Sink;
-use log::trace;
+use tracing::trace;
```

Comment:
it might be nice to also change all of the logs in this module to use structured `tracing` fields wherever possible?

Reply:
I think there is only one that has any field, fixing it!